src/hooks/useWorkoutSession.js

```
@@ -1,5 +1,6 @@
 import debounce from "lodash/debounce";
 import { useState, useEffect, useCallback } from "react";
+import { useState, useEffect, useCallback } from "react";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useState'. Consider to delete it or rename it.

'useState' is defined here:


(lint/suspicious/noRedeclare)